src/components/Text/Text.module.css

```diff
@@ -5,8 +5,78 @@
 \*------------------------------------*/
 
 /**
- * 1) 
+ * 1) h1, h2, h3, h4, h5, h6
+ * 2) Styles are either controlled by as or size declaration
  */
-/* stylelint-disable-next-line block-no-empty */
-.text {
+.text,
+.text--body, 
+.text--md {
+  @mixin reset;
+  
+  @mixin eds-theme-typography-body-text;
+}
+
+/**
+ * Sizes
+ */
+.text--lg {
+  @mixin eds-theme-typography-body-text-lg;
+}
+.text--sm {
+  @mixin eds-theme-typography-body-text-sm;
+}
+.text--xs {
+  @mixin eds-theme-typography-body-text-xs;
+}
+.text--caption {
+  @mixin eds-theme-typography-caption-text-md;
+}
+.text--overline {
+  @mixin eds-theme-typography-kicker;
+}
+
+/**
+ * Color
+ */
+.text--alert {
+  color: var(--eds-theme-color-text-utility-error);
+}
+.text--base {
+  color: var(--eds-theme-color-text-neutral-default);
+}
+.text--brand {
+  /* TODO: replace with tier 2 token */
+  color: var(--eds-color-brand-grape-700);
+}
+.text--info {
+  /* TODO: doesn't exist, replace with tier 2 token */
+  color: var(--eds-color-other-blue-700);
+}
+.text--inherit {
+  color: inherit;
+}
+.text--neutral {
+  color: var(--eds-theme-color-text-neutral-default);
+}
+.text--success {
+  color: var(--eds-theme-color-text-utility-success);
+}
+.text--warning {
+  color: var(--eds-theme-color-text-utility-warning);
+}
+.text--white {
+  /* TODO: replace with tier 2 token */
+  color: var(--eds-color-neutral-white);
+}
+
+/**
+ * Weights
+ */
+.text--normal-weight {
+  /* TODO: replace with tier 2 token */
+  font-weight: var(--eds-font-weight-medium);
+}
+.text--bold-weight {
+  /* TODO: replace with tier 2 token */
+  font-weight: var(--eds-font-weight-bold);
```

Comment:
Please inform me if these TODO's are correct, haven't found the correct ones for typography